mm2src/mm2_main/tests/docker_tests/tendermint_tests.rs

```diff
@@ -712,6 +712,52 @@ fn test_tendermint_add_delegation() {
     log!("Send raw tx {}", serde_json::to_string(&send_raw_tx).unwrap());
 }
 
+#[test]
+fn test_tendermint_remove_delegation() {
```

Comment:
Self note: may need to lock these tests to avoid account sequence number race conditions